pennylane/qnode_new/decorator.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+QNode decorator.
+"""
+from functools import lru_cache
+
+from .cv import CVQNode
+from .jacobian import JacobianQNode
+from .qnode import QNode as BaseQNode
+from .qubit import QubitQNode
+
+
+def QNode(
+    func, device, *, interface="autograd", mutable=True, differentiable=True, properties=None
+):

```

Comment:
Maybe it would be better when it comes to readability to organize the signature as follows (breaking the list of arguments):
```python
def QNode(func, device, *, interface="autograd",
        mutable=True, differentiable=True, properties=None):
```

Reply:
This was `black`'s doing 😆  I don't like it either, but am inclined to keep it as is (otherwise we lose the main benefit of using black).